src/Tooling/Bit.Tooling.Templates/TodoTemplate/Docs/Getting-Started.md

```diff
@@ -0,0 +1,78 @@
+# Getting Started
+
+This document aimed to create and run a Bit-Platform project in a short period. It is assumed that you, as the developer, are familiar with the development prerequisites that follow.
+
+## Development prerequisites
+
+- C# as the main development language.
+- [Asp.net core blazor](https://docs.microsoft.com/en-us/aspnet/core/blazor/?view=aspnetcore-6.0) as main development Back-End and Fron-End framework
+- [CSS ](https://www.google.com/url?sa=t&amp;rct=j&amp;q=&amp;esrc=s&amp;source=web&amp;cd=&amp;cad=rja&amp;uact=8&amp;ved=2ahUKEwji-KOu0pj4AhWwm_0HHeZQDzoQFnoECAgQAQ&amp;url=https%3A%2F%2Fwww.w3schools.com%2Fcss%2F&amp;usg=AOvVaw0Xtbw_GBAChsgvZNkPLVGb)&amp; [Sass ](https://www.google.com/url?sa=t&amp;rct=j&amp;q=&amp;esrc=s&amp;source=web&amp;cd=&amp;cad=rja&amp;uact=8&amp;ved=2ahUKEwjvgoO60pj4AhUCi_0HHVmXBMkQFnoECAgQAQ&amp;url=https%3A%2F%2Fsass-lang.com%2F&amp;usg=AOvVaw0p_IRgLEbIPRGWtlW7Wph8)as stylesheet
+- [Ef Core](https://docs.microsoft.com/en-us/ef/core/) as ORM to communicate with the database
+- [Asp.Net Identity](https://docs.microsoft.com/en-us/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity) with [JWT ](https://www.c-sharpcorner.com/article/jwt-authentication-and-authorization-in-net-6-0-with-identity-framework/)supporting for handling Authentication
+
+## Pre Requirements tools
+
+- Microsoft Visual Studio 2022 - Preview Version 17.3.0 Preview 1.0 or higher
+- Asp.net and web development VS workload
+- .Net Multi-Platform App UI development (for mobile app development target) VS workload
+
+## Setup envirnoment
+vs extention...
+
+
+## Create project
+
+Clone the [TodoTemplate](https://github.com/bitfoundation/bitplatform/tree/develop/src/Tooling/Bit.Tooling.Templates/TodoTemplate) from the Bit-Platform Github repository
```

Comment:
Use dotnet new approach based on assumption that we've `Bit.Tooling.Templates.TodoTemplate` nuget package

Reply:
Ok added